### PR TITLE
Graphqxl blog post

### PR DESCRIPTION
**Pull request by gabotechs**
Would be nice to have a better cover image that matches the format and styling of the other blog posts you have, maybe someone can help me with that? 

About the blog post itself, I am not very familiar with the structure that you usually follow, the length or even the average audience that would see this, so I have kept it very technical and more or less short. Please don't hesitate about making suggestions in these aspects.

### CODE REVIEW

File: website/pages/blog/graphqxl-language.mdx [hunk 1]
```diff
@@ -0,0 +1,190 @@
+---
+title: GraphQXL language
```
Comment (Urigo) [inline website/pages/blog/graphqxl-language.mdx:2]:
```suggestion
title: GraphQXL - The missing GraphQL Language Extension
```

File: website/pages/blog/graphqxl-language.mdx [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+## The Need of a New Language
+
+GraphQL is a really nice language for writing API schemas, but it tends to fall short on
+features as it is not a programming language (and it is not meant to be).
+
+With a code-first approach it is very easy to autogenerate repetitive graphql
+code as it is the programming language of choice the one that does all the hard work,
+but with a schema-first approach the limits are set by the capabilities of vanilla GraphQL.
+
+GraphQXL is directed for people that want to keep using a schema-first approach but also
```
Comment (Urigo) [inline website/pages/blog/graphqxl-language.mdx:23]: maybe worth mentioning here some of the benefits of schema first approach?
maybe not, not sure...

**Review by Urigo — APPROVED** [review]

File: website/pages/blog/graphqxl-language.mdx [hunk 3]
```diff
@@ -0,0 +1,190 @@
+---
+title: GraphQXL language
+tags: [graphql]
+authors: [gabotechs]
+date: 2022-11-30
+description: GraphQXL, a new language for building big and scalable GraphQL server-side schemas
+image: /blog-assets/graphqxl-language/cover.svg
+---
+
+When building GraphQL APIs following a schema-first approach, as the project goes big,
```
Comment (enisdenjo) [inline website/pages/blog/graphqxl-language.mdx:10]:
```suggestion
When building GraphQL APIs following a schema-first approach and the project goes big,
```

File: website/pages/blog/graphqxl-language.mdx (hunk 1 above)
Comment (n1ru4l) [reply website/pages/blog/graphqxl-language.mdx:2]: I would add a question mark to make it less "dictating"

Comment (n1ru4l) [reply website/pages/blog/graphqxl-language.mdx:2]:
```suggestion
title: GraphQXL language - The missing GraphQL Language Extension?
```

File: website/pages/blog/graphqxl-language.mdx [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+people usually find themselves repeating almost identical structures across the schema and
+copy-pasting common fields across the different types and inputs.
+
+## The Need of a New Language
+
+GraphQL is a really nice language for writing API schemas, but it tends to fall short on
```
Comment (n1ru4l) [inline website/pages/blog/graphqxl-language.mdx:16]:
```suggestion
GraphQL SDL is a really nice language for writing API schemas, but it tends to fall short on
```

**Review by n1ru4l — APPROVED** [review]
Sorry for the late review, I just have a small paragraph suggestion and headline adjustment 😇

**Comment by n1ru4l** [comment]
Regarding the cover/thumbnail we should make sure both exist and have the same style/dimensions as existing blog posts on https://the-guild.dev/blog

File: website/pages/blog/graphqxl-language.mdx [hunk 5]
```diff
@@ -0,0 +1,201 @@
+---
+title: GraphQXL - The missing GraphQL Language Extension?
+tags: [graphql]
+authors: [gabotechs]
+date: 2022-11-30
```
Comment (n1ru4l) [inline website/pages/blog/graphqxl-language.mdx:5]: need to adjust the date before publishing. @Urigo @dotansimha lets publish it today?

**Comment by Urigo** [comment]
@n1ru4l yes let's publish today!
I will in flights, can you take care of it?  maybe also tweet?

File: website/pages/blog/graphqxl-language.mdx (hunk 5 above)
Comment (n1ru4l) [inline website/pages/blog/graphqxl-language.mdx:5]:
```suggestion
date: 2022-12-12
```

**Pull request merged**